src/components/repository/NewRepositoryForm.vue

```diff
@@ -1,21 +1,21 @@
 <template>
-  <form @submit.prevent="onSubmit()">
-    <loading v-if="!formSchema" />
-    <form-generator
-      v-if="formSchema"
-      :schema="formSchema"
-      v-model="data"
-      :errors="errors"
-      class="field" />
-    <div class="field">
-      <div class="control has-text-centered">
-        <button
+  <div class="new-repository-form">
+    <form @submit.prevent="onSubmit()">
+      <loading v-if="!formSchema" />
+      <form-generator
+        v-if="formSchema"
+        :schema="formSchema"
+        v-model="data"
+        :errors="errors" />
+      <div class="new-repository-form__create-bot-button">
```

Comment:
Move text-center (App.vue) to BH classes and use here. Remove this class definition.